examples/example-react-native-app/utils/useAuthorization.tsx

```diff
@@ -11,24 +11,30 @@ import {toUint8Array} from 'js-base64';
 import {useCallback} from 'react';
 import useSWR from 'swr';
 
+type Account = Readonly<{
```

Comment:
Can you elaborate on the thinking behind this type? From a chain-agnostic POV, it makes sense; addresses and public keys aren't necessarily the same. However, this is in a Solana network example, where we can safely make that assumption. I'm not opposed to it, since it does encode a valuable distinction in an example that is likely to be copied and reused, just curious to hear your thinking.

Reply:
For sure!

At some point, this app is going to need to call `signMessage` and when it does, the API will demand that it specifies an `account`. If I throw away the base64 encoded account string, I’ll need to reconstruct it from the pubkey later.

Secondly: at some point the user will switch accounts using a combo box. If I’ve thrown away the base64 encoded account strings, I’ll have to reconstruct the selected one from the pubkey in order to make the choice of account to switch to.